salt/roster/__init__.py

```diff
@@ -104,6 +104,7 @@ def targets(self, tgt, tgt_type):
             except salt.exceptions.SaltRenderError as exc:
                 log.error('Unable to render roster file: {0}'.format(exc))
             except IOError as exc:
+                log.error('Can\'t access roster for backend {0}: {1}'.format(back, exc))
```

Comment:
Salt conventions do indicate use of single quotes:

> In Salt, all strings use single quotes unless there is a good reason not to. This means that docstrings use single quotes, standard strings use single quotes etc.:

So, this boils down on whether to use double quotes as an exception here and why that is better than a backslash. Can you point me to any resource indicating why double quotes is preferred to a backslash in this case, or are you just pushing your personal preference?